spec/models/on/update_spec.rb

```diff
@@ -5,9 +5,10 @@
 
 module On
   RSpec.describe Update, type: :model, versioning: true do
+    let(:record) { described_class.create(name: "Alice") }
```

Comment:
Please do not use `let`. I believe it makes tests harder to understand, especially when it's too far away, lexically, from the test.

Reply:
Reading this from 2020, and I completely agree. It can be useful for dynamic values to DRY up tests but if it's overused, it becomes a nightmare. Additionally, the `let` block is only executed once it is called so that alone can be confusing and have detrimental side effects with tests.